### PR TITLE
836: Disambiguation of Spanish locales

### LINKED ISSUE #836: Español de Costa Rica is missing part of label
`wp_get_available_translations()` in WP Core shows "Español" as the native name for both `es_CR` (Costa Rica) and `es_ES`. This is only possible to see by looking at the markup. I believe that the first should be something like "Español de Costa Rica" (or whatever the suitably localised version would be).

https://github.com/GlotPress/GlotPress-WP/blob/a278bb001d99bc96c2e21e20002958a7c7d3d222/locales/locales.php#L693

espanol-label-missing

### PR DESCRIPTION
**Pull request by tobifjellner**
+added 639-3 for all Spanish locales

Fixes #836.

### CODE REVIEW

**Comment by tobifjellner** [comment]
Question: Would we break anything if you at the same time change the locale code es_CR to es_cr?

File: locales/locales.php [hunk 1]
```diff
@@ -647,8 +647,10 @@ public function __construct() {
 
 		$es = new GP_Locale();
 		$es->english_name = 'Spanish (Spain)';
-		$es->native_name = 'Español';
+		$es->native_name = 'Español de España';
```
Comment (ocean90) [inline locales/locales.php]: Please leave this as it is.

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Comment by ocean90** [comment]
@tobifjellner No, please change it to `es_cr`.

**Review by swissspidy — APPROVED** [review]
Looks good to me.

**Review by ocean90 — APPROVED** [review]

**Pull request merged**